apps/parsley/src/components/LogRow/SectionHeader/SubsectionControls.tsx

```diff
@@ -0,0 +1,45 @@
+import Button from "@leafygreen-ui/button";
+import { useLogContext } from "context/LogContext";
+
+const SubsectionControls: React.FC<{ functionID: string }> = ({
+  functionID,
+}) => {
+  const { sectioning } = useLogContext();
+  const { sectionState, toggleAllCommandsInFunction } = sectioning;
+  const showExpandButton =
+    sectionState &&
+    (Object.values(sectionState[functionID].commands).some(
+      (commandID) => !commandID.isOpen,
+    ) ||
+      !sectionState[functionID].isOpen);
+  const showCloseButton =
+    sectionState &&
+    Object.values(sectionState[functionID].commands).some(
+      (commandID) => commandID.isOpen,
+    ) &&
+    sectionState[functionID].isOpen;
+  return (
+    <>
+      {showExpandButton && (
+        <Button
+          data-cy="open-subsections-btn"
+          onClick={() => toggleAllCommandsInFunction(functionID, true)}
+          size="xsmall"
+        >
+          Open subsections
```

Comment:
I noticed this doesn't match the [design](https://www.figma.com/proto/WmuwqR5h1mSEHE3XqROdRl/EXPO-2829---Sections?node-id=308-7470&t=2TljZyU1eW1MJUke-1) exactly, just wanted to confirm it was intentional

<img width="346" alt="image" src="https://github.com/user-attachments/assets/8f110399-709e-4c92-ae44-3a83240acae0">

Reply:
This is intentional. The buttons on the section headers are not in the design but have been reviewed by Will.